### PR TITLE
Commonize favorite session items and speaker session items #383

### LINKED ISSUE #383: [Assigned] Commonize favorite session items and speaker session items
## Overview (Required)
- We can use simplify variable in SpeechSessionItem, So probably we don't need item_favorite_session.xml and FavoriteSessionItem

### PR DESCRIPTION
**Pull request by citrous**
Commonize favorite session items and speaker session items

## Issue
- close #383 

## Overview (Required)
- Using SpeechSessionItem for FavoriteSessionsSecsion instead of FavoriteSessionItem

## Screenshot
before
device-2018-01-21-193056

after
device-2018-01-21-201410

### CODE REVIEW

**Comment by takahirom** [comment]
Could you paste current Favorite screenshot？

**Comment by citrous** [comment]
Sorry, I fixed margin and upload screenshots.

**Comment by takahirom** [comment]
Thanks!👍

**Comment by takahirom** [comment]
Looks good!

File: app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/item/FavoriteSessionsSection.kt [hunk 1]
```diff
@@ -36,12 +39,7 @@ class FavoriteSessionsSection : Section() {
 
         var item = getItemOrNull(position) ?: return null
         item = item as? SpeechSessionItem ?: getItemOrNull(position + 1) ?: return null
-        return when (item) {
-            is SessionItem -> {
-                item.session.dayNumber
-            }
-            else -> null
-        }
+        return (item as? SpeechSessionItem)?.session?.dayNumber
```
Comment (takahirom) [inline app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/item/FavoriteSessionsSection.kt]: 👍

**Pull request merged**